tanner/create_tables.py

```diff
@@ -0,0 +1,69 @@
+import asyncio
+
+
+class CreateTables:
+    @staticmethod
+    async def create_data_tables(pg_client):
+        """Create all the required tables in
+            the postgres database
+
+        Arguments:
+            pg_client {aiopg.pool.Pool}
+        """
+        async with pg_client.acquire() as conn:
+            async with conn.cursor() as cur:
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "sessions" (
+                    "id" UUID PRIMARY KEY,
+                    "sensor_id" TEXT NOT NULL,
+                    "peer.ip" INET NOT NULL,
+                    "peer.port" INT NOT NULL,
+                    "location.country" TEXT NULL,
+                    "location.country_code" TEXT NULL,
+                    "location.city" TEXT NULL,
+                    "location.zip_code" INT NULL,
+                    "user_agent" TEXT NOT NULL,
+                    "start_time" TIMESTAMP DEFAULT NOW(),
+                    "end_time" TIMESTAMP DEFAULT NOW(),
+                    "rps" FLOAT NULL,
+                    "approx_time_between_requests" FLOAT NULL,
+                    "accepted_paths" INT NULL,
+                    "errors" INT NULL,
+                    "hidden_links" INT NULL,
+                    "referer" TEXT NULL,
+                    "possible_owners.user" FLOAT NULL,
+                    "possible_owners.type" FLOAT NULL,
+                    "possible_owners.tool" FLOAT NULL,
+                    "possible_owners.crawler" FLOAT NULL
+                    )
+                """
+                )
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "paths" (
+                    "sess_uuid" UUID REFERENCES sessions(id),
+                    "path" TEXT NOT NULL,
+                    "created_at" TIMESTAMP NOT NULL,
+                    "response_status" INT NOT NULL,
+                    "attack_type" INT NOT NULL
+                )
+                """
+                )
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "cookies" (
+                    "sess_uuid" UUID REFERENCES sessions(id),
+                    "key" TEXT NULL,
+                    "value" TEXT NULL
+                )
+                """
+                )
+                await cur.execute("comment on column sessions.rps is 'requests per second'")
+                await cur.execute("CREATE INDEX ON sessions(sensor_id)")
```

Comment:
why are you adding those indexes?

Reply:
to speed up the process of searching. Like if we want to take out all the sessions having a single snare_id then we can do that.